docs/proposals/002-api-proposal/proposal.md

```diff
@@ -80,101 +80,99 @@ The API design is based on these axioms:
 - The MVP will heavily assume requests are done using the OpenAI spec, but open to extension in the future
 - The Gateway should route in a way that does not generate a queue of requests at the model server level
 
-The [PoC](https://youtu.be/NUBZg_uqqXk?si=v681EeYdGUGEVqQQ&t=1458) was focused on lower-level scheduling. And the API follows that similar logic, which lead to the proposal of the **LLMServerPool**.
+The [PoC](https://youtu.be/NUBZg_uqqXk?si=v681EeYdGUGEVqQQ&t=1458) was focused on lower-level scheduling. And the API follows that similar logic, which lead to the proposal of the **InferencePool**.
 
-### LLMServerPool
+### InferencePool
 
-The LLMServerPool at its core is a logical grouping of compute, expressed in the form of Pods (typically model servers), akin to a K8s Service. The LLMServerPool would deploy its own routing, and offer administrative configuration to the Platform Admin. 
+The InferencePool at its core is a logical grouping of compute, expressed in the form of Pods (typically model servers), akin to a K8s Service. The InferencePool would deploy its own routing, and offer administrative configuration to the Platform Admin. 
 
- It is expected for the LLMServerPool to:
- - Enforce fair consumption of resources across competing services
+ It is expected for the InferencePool to:
+ - Enforce fair consumption of resources across competing workloads
  - Efficiently route requests across shared compute (as displayed by the PoC)
  
-It is _not_ expected for the LLMServerPool to:
+It is _not_ expected for the InferencePool to:
  - Enforce any common set of adapters or base models are available on the Pods
  - Manage Deployments of Pods within the Pool
  - Manage Pod lifecycle of pods within the pool 
 
-Additionally, any Pod that seeks to join a LLMServerPool would need to support a protocol, defined by LLM Instance Gateway, to ensure the Pool has adequate information to intelligently route requests.
+Additionally, any Pod that seeks to join an InferencePool would need to support a protocol, defined by this project, to ensure the Pool has adequate information to intelligently route requests.
 
-### LLMService
+### InferenceModel
 
-A LLMService allows the LLM Service Owner to define:
+An InferenceModel allows the Inference Workload Owner to define:
 - Which LoRA adapter(s) to consume 
-  - LLMService allows for traffic splitting between adapters _in the same LLMServerPool_ to allow for new LoRA adapter versions to be easily rolled out 
-- SLO objectives for the LLMService
-- The Pools this LLMService is relevant to 
+  - InferenceModel allows for traffic splitting between adapters _in the same InferencePool_ to allow for new LoRA adapter versions to be easily rolled out 
+- SLO objectives for the InferenceModel
+- The Pools this InferenceModel is relevant to 
 
 ### Spec
 
-**LLMService**
+**InferenceModel**
```

Comment:
nit: Above we introduce InferencePool first, so let's change the order of the spec to have InferencePool as the first